src/wasm.h

```diff
@@ -1610,6 +1614,17 @@ class Module {
     }
     importsMap.erase(name);
   }
+
+  void removeImportsWithSubstring(Name name) {
```

Comment:
Done in https://github.com/WebAssembly/binaryen/pull/910/commits/bdaa866c5b4c46eb0a12f64aed681b1b547199f5